tests/tests_nn/test_unet_2d.py

```diff
@@ -56,6 +63,15 @@ def test_unet_2d(shape, num_filters, num_pool_layers, skip, normalized):
     data = create_input(shape + [2]).cpu()
     sens = create_input(shape + [2]).cpu()
 
+    model.eval()
+
     out = model(data, sens)
 
+    if openvino_available:
+        ov_model = OpenVINOModel(model)
+        # ov_out = ov_model(masked_kspace=data, sensitivity_map=sens)
+        ov_out = ov_model(masked_kspace=data)
```

Comment:
Model launch should be equal: `ov_out = ov_model(data, sens)`

Reply:
Yes